keyboards/akb/raine/keymaps/MehAdviceGuy/keymap.c

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2019 Elliot Powell
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define ALT_TAB LALT(KC_TAB)
```

Comment:
This isn't used, and can be removed. 

```suggestion
```

Reply:
Hello Drashna,
Thank you for pointing this out to me! 
I have removed this in commit "7d9d5d2184ff013c51213b68ef1893c0af0d420a"

Best regards